bodhi/server/models/models.py

```diff
@@ -1400,6 +1400,18 @@ def get_pushed_color(self):
             color = '#00ff00'  # green
         return color
 
+    def obsolete_if_unstable(self, db):
+        """
+        If an update with pending status(autopush enabled) reaches unstable karma
+        threshold, make sure it gets obsoleted.
+        """
+        if self.autokarma and self.status is UpdateStatus.pending and self.request is UpdateRequest.testing \
+            and self.unstable_karma not in (0, None) and self.karma <= self.unstable_karma:
```

Comment:
Can you add one more level of indentation to this line so that it's visually separated from the code below?